initial_angle: 0.0